webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -559,4 +561,84 @@ test.describe('Experiment List', () => {
       });
     });
   });
+
+  test.describe('Row Actions', () => {
+    let destinationProject: V1Project;
+    let destinationWorkspace: V1Workspace;
+    let experimentId: number;
+
+    // create a new project, workspace and experiment
+    test.beforeAll(
+      async ({
+        backgroundApiProject,
+        backgroundApiWorkspace,
+        newProject: {
+          response: { project },
+        },
+      }) => {
+        destinationWorkspace = (
+          await backgroundApiWorkspace.createWorkspace(backgroundApiWorkspace.new())
+        ).workspace;
+        destinationProject = (
+          await backgroundApiProject.createProject(
+            project.workspaceId,
+            backgroundApiProject.new({ projectProps: { workspaceId: project.workspaceId } }),
+          )
+        ).project;
+
+        const expId = Number(
+          detExecSync(
+            `experiment create ${fullPath('examples/tutorials/mnist_pytorch/adaptive.yaml')} --paused --project_id ${project.id}`,
+          ).split(' ')[2],
+        ); // returns in the format "Created experiment <exp_id>"
+
+        if (!Number.isNaN(expId)) experimentId = expId;
+      },
+    );
+
+    // cleanup
+    test.afterAll(async ({ backgroundApiProject, backgroundApiWorkspace }) => {
+      if (experimentId !== undefined) {
+        detExecSync(`experiment kill ${experimentId}`);
+        detExecSync(`experiment delete ${experimentId} --y`);
+      }
+
+      await backgroundApiProject.deleteProject(destinationProject.id);
+      await backgroundApiWorkspace.deleteWorkspace(destinationWorkspace.id);
+    });
+
+    test('move experiment', async ({
+      newWorkspace: {
+        response: { workspace },
+      },
+    }) => {
+      if (experimentId === undefined) return;
+
+      const newExperimentRow =
+        await projectDetailsPage.f_experimentList.dataGrid.getRowByColumnValue(
+          'ID',
+          experimentId.toString(),
+        );
+
+      const menuMove = await newExperimentRow.experimentActionDropdown.open();
+
+      await menuMove.menuItem('Move').pwLocator.click();
+      await menuMove.moveModal.destinationWorkspace.selectMenuOption(workspace.name);
+      await menuMove.moveModal.destinationProject.pwLocator.waitFor({ state: 'visible' });
+      await menuMove.moveModal.destinationProject.selectMenuOption(destinationProject.name);
+      await menuMove.moveModal.footer.submit.pwLocator.click();
+      await menuMove.moveModal.pwLocator.waitFor({ state: 'hidden' });
+
+      await newExperimentRow.pwLocator.waitFor({ state: 'hidden' });
+
+      await projectDetailsPage.gotoProject(destinationProject.id);
+      const grid = projectDetailsPage.f_experimentList.dataGrid;
+      await grid.setColumnHeight();
+      await grid.headRow.setColumnDefs();
+      const newProjectRows = await projectDetailsPage.f_experimentList.dataGrid.filterRows(() =>
+        Promise.resolve(true),
+      );
+      await expect(newProjectRows.length).toBe(1);
```

Comment:
shouldn't matter in practice, but more accurate to find row by id, instead of just counting all rows?